.github/workflows/template-sync-app.yml

```
@@ -0,0 +1,31 @@
+        uses: actions/checkout@v4
+        with:
+          token: ${{ steps.generate_token.outputs.token }}
+
+      - name: actions-template-sync
+        uses: AndreasAugustin/actions-template-sync@v2
+        with:
+          github_token: ${{ steps.generate_token.outputs.token }}
+          source_repo_path: VilnaCRM-Org/php-service-template
+          upstream_branch: main
+          pr_labels: template_sync
+          is_force_push_pr: true
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)